spec/classes/collectd_init_spec.rb

```diff
@@ -4,7 +4,9 @@
   on_supported_os.each do |os, facts|
     context "on #{os} " do
       let :facts do
-        facts
+        facts.merge(
+          python_dir: '/usr/local/lib/python2.7/dist-packages'
```

Comment:
can you move the fact to https://github.com/voxpupuli/puppet-collectd/blob/master/spec/default_module_facts.yml?